File: Finance_calc/app/src/main/java/com/example/finance_calc/MainActivity.java
package com.example.finance_calc;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.example.first_hackathon.R;
import com.example.finance_calc.activities.CreditActivity;
import com.example.finance_calc.activities.DepositActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //get main buttons
        Button btnDeposit = findViewById(R.id.btn_deposit);
        Button btnCredit = findViewById(R.id.btn_credit);

        //add click listeners to buttons
        btnDeposit.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getApplicationContext(), DepositActivity.class);
                startActivity(intent);
            }
        });

        btnCredit.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getApplicationContext(), CreditActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: Film_search/app/src/main/java/com/example/film_search/MainActivity.kt
package com.example.film_search

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.film_search.fragments.InfoFragment
import com.example.film_search.fragments.MovieListFragment
import com.example.film_search.listeners.interfaces.ActivityNavigationListeners
import com.example.film_search.utils.HelpUtils

class MainActivity : AppCompatActivity(),
    ActivityNavigationListeners {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        savedInstanceState.let {
            HelpUtils.replaceFragment(
                this,
                MovieListFragment.newInstance(),
                MovieListFragment.TAG,
                false
            )
        }
    }

    override fun showSingleMoviePage(movieId: String) {
        HelpUtils.replaceFragment(
            this,
            InfoFragment.newInstance(movieId),
            InfoFragment.TAG,
            true
        )
    }
}